lib/core/versions/latest/AnchorFile.ts

```diff
@@ -36,7 +36,8 @@ export default class AnchorFile {
 
     let anchorFileDecompressedBuffer;
     try {
-      anchorFileDecompressedBuffer = await Compressor.decompress(anchorFileBuffer);
+      const estimatedDecompressedFileSize = ProtocolParameters.maxAnchorFileSizeInBytes * Compressor.estimatedDecomporessionMultiplier;
```

Comment:
Nit: maybe update the variable name to be `estimatedDecompressedFileSizeInBytes`? Same for the other uses.

Reply:
Renamed to `maxAllowedDecompressedSizeInBytes`.